packages/graphql/src/classes/CallbackBucket.ts

```diff
@@ -47,9 +47,11 @@ export class CallbackBucket {
         await Promise.all(
             this.callbacks.map(async (cb) => {
                 const callbackFunction = (this.context?.callbacks as Neo4jGraphQLCallbacks)[cb.functionName] as (
-                    args?: Record<string, unknown>
+                    parent?: Record<string, unknown>,
+                    _args?: Record<string, unknown>,
```

Comment:
Sry for not knowing this, does the leading underscore mean/indicate that the variable is unused?

Reply:
Usually that shuts up our eslint rule of "unused variable" as it is understood that it is required for the function to work, but not used inside. Not sure in this case

Reply:
I can just call this `args` if you like - I did this more for us just to show that it's unused.